src/objects/xnorGate.js

```diff
@@ -0,0 +1,14 @@
+import { XorGate } from "xorGate.js";
```

Comment:
I tried swapping in the XnorGate here and got a TypeError in the console:

```
index.html:1 Uncaught TypeError: Failed to resolve module specifier "xorGate.js". Relative references must start with either "/", "./", or "../".
```

When I added a './' to the file path here, I then got a similar error for the import statement in `xorGate.js` for `twoInputGate.js`, and when I fixed that one I saw a separate error related to how that class was being imported:

```
Uncaught SyntaxError: The requested module './twoInputGate.js' does not provide an export named 'TwoInputGate'
```
I removed the curly braces around the import in `xorGate.js` and saw an error in the way I was importing `xnorGate.js` in the challenge mode scene.

I'd recommend going through these classes to make sure they all `export default class`, which will make importing them easier (so we don't have to use named imports for some and not others), and then also testing to make sure they're all wired up correctly (i.e. that the file paths are correct).

Reply:
I wasn't paying attention to how I was exporting the gates. This will be fixed by removing the curly Brackets.